repositories.go

```diff
@@ -14,19 +20,45 @@ type Repositories struct {
 	repositories
 }
 
+func safelyConvertToURIParameters(ro *RepositoriesOptions) string {
+	params := ""
```

Comment:
disclaimer: :wave: just passing through the repo, saw this PR and hope this gets merged!

instead of building params manually and maintaining consistency with &/encoding's, you may find some clarity and convenience in using `url.Values{}` from [net/url](https://golang.org/pkg/net/url/#Values). eg.
```
params := url.Values{}
...
    params.Add("page", pageString)
...
    params.Add("pageLen", pageLenString)
... 
return params.Encode()
```

Reply:
Totally forgot the Values object. I was looking for an encode method that worked on a map. Thanks!